miqa/core/models/frame.py

```diff
@@ -38,3 +39,15 @@ def size(self) -> int:
     @property
     def experiment(self) -> Experiment:
         return self.scan.experiment
+
+    @property
+    def is_in_s3(self) -> bool:
+        return self.raw_path.startswith('s3://')
+
+    @property
+    def s3_download_url(self) -> Optional[str]:
+        if not self.is_in_s3:
+            return None
+        bucket, key = self.raw_path.strip()[5:].split('/', maxsplit=1)
+        client = boto3.client('s3')
```

Comment:
Does this need auth at all?

Reply:
The auth will come through the environment variables that will be set in the runtime environment